.gitignore

```diff
@@ -0,0 +1,144 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
Whoa, this file is significant. Surely we don't need all this?

Reply:
https://github.com/github/gitignore/blob/master/Python.gitignore

This is based on the Python .gitignore template.

Reply:
That seems like overkill. can we keep it to the stuff we actually need, please?

Reply:
Removed the tooling specific entries in 97698c7

Reply:
Alright, thank you.